src/IntlPhoneInput.js

```diff
@@ -18,11 +18,11 @@ import CloseIconAndroid from './close.android.png';
 
 import data from './Countries';
 
-export default class IntlPhoneInput extends React.Component {
+class IntlPhoneInput extends React.Component {
   constructor(props) {
     super(props);
     const defaultCountry =
-        data.filter((obj) => obj.code === props.defaultCountry)[0] || data.filter((obj) => obj.code === "TR")[0];
+         data.filter((obj) =>  props.phoneNumber.includes(obj.dialCode))[0] || data.filter((obj) => obj.code === props.defaultCountry)[0] || data.filter((obj) => obj.code === 'US')[0];
```

Comment:
Default country shouldn't be configurable via a prop?

Reply:
Yeap you can set default country via prop